reference/markdown/Markdown-schema.md

```diff
@@ -0,0 +1,147 @@
+As of version 5.0, the _LwDITA_ plug-in provides the `MarkdownReader` class that supports a new `$schema` key in the YAML front matter block. This key can be used to define parsing options in topic files for more control over how Markdown content is converted to DITA.
+
+```markdown
+---
+$schema: urn:oasis:names:tc:dita:xsd:concept.xsd
+---
+
+# Concept title
+
+Shortdesc content.
+
+Body content.
+```
+
+The `$schema` value is URI that is mapped to a parser configuration. This defined which the document should be parsed, i.e. which Markdown flavour it uses. Markdown schema definition is similar to XML document type declaration or `<?xml-model?>` processing instruction where the document defines how it should be optionally validated.
+
+The `$schema` key must be the first key in the YAML header.
+
+## Built-in schemas
+
+### DITA topic
+
+- `urn:oasis:names:tc:dita:xsd:topic.xsd`
+- `urn:oasis:names:tc:dita:xsd:topic.rng`
+
+### DITA concept
+
+- `urn:oasis:names:tc:dita:xsd:concept.xsd`
+- `urn:oasis:names:tc:dita:xsd:concept.rng`
+
+### DITA task
+
+- `urn:oasis:names:tc:dita:xsd:task.xsd`
+- `urn:oasis:names:tc:dita:xsd:task.rng`
+
+### DITA reference
+
+- `urn:oasis:names:tc:dita:xsd:reference.xsd`
+- `urn:oasis:names:tc:dita:xsd:reference.rng`
+
+### DITA map
+
+- `urn:oasis:names:tc:dita:xsd:map.xsd`
+- `urn:oasis:names:tc:dita:xsd:map.rng`
+
+### Lightweight DITA topic extended profile
+
+- `urn:oasis:names:tc:mdita:xsd:topic.xsd`
+- `urn:oasis:names:tc:mdita:rng:topic.rng`
+- `urn:oasis:names:tc:mdita:extended:xsd:topic.xsd`
+- `urn:oasis:names:tc:mdita:extended:rng:topic.rng`
+
+### Lightweight DITA topic core profile
+
+- `urn:oasis:names:tc:mdita:core:xsd:topic.xsd`
+- `urn:oasis:names:tc:mdita:core:rng:topic.rng`
+
+## Custom schema configuration
```

Comment:
As I said before, this should be a separate topic.

Reply:
Resolved in 2864c89.